venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import openreview
+from tqdm import tqdm
+
+"""
+OPTIONAL SCRIPT ARGUMENTS
+
+    baseurl -  the URL of the OpenReview server to connect to (live site: https://openreview.net)
+    username - the email address of the logging in user
+    password - the user's password
+
+"""
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+# Iterate through all commitment submissions 
+notes = openreview.tools.iterget_notes(client,invitation='aclweb.org/ACL/2022/Conference/-/Commitment')
+for note in notes:
+    # For each, retrieve old paper from link
+    link = note.content['paper_link']
+    forum = link.split('=')[1]
+    old_note = client.get_note(client.get_note(forum).original)
+# Create JSON for new note with readers
+    JSON = {
```

Comment:
Instead of using a JSON, why don't you use the Python class directly?

note = openreview.Note(
  invitation="aclweb.org/ACL/2022/Conference/-/Submission",
  readers=["aclweb.org/ACL/2022/Program_Chairs","aclweb.org/ACL/2022"],
....
  
)